api/envoy/config/rbac/v3/rbac.proto

```diff
@@ -310,3 +310,29 @@ message Principal {
     Principal not_id = 8;
   }
 }
+
+// Action defines the result of allowance or denial when a request matches the matcher.
+message Action {
+  // The name indicates the matcher name.
```

Comment:
nit: I think this is the effective policy name, not matcher name.

Reply:
Yes, I will change that.